backend/inspirehep/matcher/validators.py

```diff
@@ -0,0 +1,16 @@
+from inspire_utils.record import get_value
+
+
+def authors_validator(author, result):
+    record_author_identifiers = author.get("ids")
```

Comment:
```suggestion
    record_author_identifiers = author.get("ids", [])
```
otherwise it will fail in the for loop